### ISSUE TITLE
[NEW] Send 'mutable-content = 1' in APN payload

### ISSUE THREAD

**Issue by cardoso** [RocketChat/Rocket.Chat#12956]
This will allow us to support end-to-end encrypted notifications & also remove markdown in the client side.

https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/ModifyingNotifications.html

**Comment by geekgonecrazy** [comment]
Looks like in raix/push it would be modified around here: https://github.com/raix/push/blob/master/lib/common/notifications.js#L30 . Just would need to make sure that its an int here instead of boolean

As well as around here: https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-lib/server/lib/PushNotification.js#L19

Not sure exactly where the logic is that talks to push gateway but if not part of the above then we'd need to change there too :)

**Comment by cardoso** [comment]
@geekgonecrazy are we not using https://github.com/RocketChat/push now in place of raix?

**Comment by geekgonecrazy** [comment]
You're right we are: https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-api/server/v1/push.js#L4

I think we did while waiting on them to include a patch.  I think they have since merged it... so not sure if the fork is still necessary or not.  @sampaiodiego would be able to tell us for sure :)
